AutoRip.js

```diff
@@ -398,62 +411,73 @@ function ripDVD(commandDataItem, outputPath) {
         colors.info(" to " + dir + "...")
     );
 
-    exec(
-      makeMKV +
+    if (mediaType === 'dvd') {
+      makeMKVCommand =
+        makeMKV +
         " -r mkv disc:" +
         commandDataItem.driveNumber +
         " " +
         commandDataItem.fileNumber +
         " " +
         '"' +
         dir +
-        '"',
-      (err, stdout, stderr) => {
-        if (stderr) {
-          console.error(
-            colors.time(moment().format("LTS")) +
-              colors.dash(" - ") +
-              colors.error(
-                "Critical Error Ripping " + colors.title(commandDataItem.title),
-                stderr
-              )
-          );
-          reject(stderr);
-        } else {
-          //console.log(colors.blue('OUTPUT', stdout)); //Outputs full log data to console after ripping (or attempting to rip) each DVD
-          var fileName = createUniqueFile(logDir, commandDataItem.title);
-
-          if (fileLog == "true") {
-            fs.writeFile(fileName, stdout, "utf8", function (err) {
-              if (err)
-                console.error(
-                  colors.time(moment().format("LTS")) +
-                    colors.dash(" - ") +
-                    colors.error(
-                      "Directory for logs does not exist. Please Create it."
-                    )
-                );
-              console.info(
+        '"';
+    } else if (mediaType === 'blu-ray') {
+      makeMKVCommand =
+        makeMKV +
+        " mkv disc:" +
+        commandDataItem.driveNumber +
+        " all " +

```

Comment:
This causes all the video files to be ripped, instead of just the biggest one. I was thinking of adding an option in the ripOrDip function to allow the user to choose at the start of ripping.

Reply:
Yea I think that's a good idea. At least for my use case, the longest file is normally best for movies... but really isn't compatible with episodic content.

Perhaps adding an option in the config file for that and then displaying the selected option on startup?

Reply:
Option added